libminifi/test/aws-tests/MockS3RequestSender.h

```diff
@@ -227,7 +227,7 @@ class MockS3RequestSender : public minifi::aws::s3::S3RequestSender {
       const Aws::Client::ClientConfiguration& client_config) override {
     head_object_request = request;
     credentials_ = credentials;
-    client_config_ = client_config;
+    client_config_ = const_cast<Aws::Client::ClientConfiguration*>(&client_config);
```

Comment:
Maybe it would be easier to make `client_config_` `const` instead of const_casting everywhere? I only found reading from the client config in the tests